common/build/build-common/README.md

```diff
@@ -6,24 +6,21 @@ See [GitHub](https://github.com/microsoft/FluidFramework) for more details on Fl
 
 ## API-Extractor Configuration
 
-This package exports a base configuration for use with [API-Extractor](https://api-extractor.com/).
+This package exports several base configurations for use with [API-Extractor](https://api-extractor.com/).
 It can be extended in your package's local configuration file like the following:
 
 ```json
-"extends": "@fluidframework/build-common/api-extractor-base.json",
+"extends": "@fluidframework/build-common/api-extractor-base.(cjs|esm).<export set>.json",
 ```
 
-### Dual Build Considerations
+Chose `cjs` or `esm` based on primary or only output.
 
-A variety of configuration files were build and named while dual build pattern was being developed and have not been rationalized for what is believed to be the final state. The import aspect is to select a set of files that generate the report once. With the current dual build pattern this means using both of the `api-extractor-base.(cjs|esm).primary.json` files and then configuration override of:
+### Export Set
```

Comment:
Might be worth explaining or linking to an explanation of what "legacy" is, and why it exists.